app/src/main/res/layout/context_menu.xml

```diff
@@ -8,11 +8,15 @@
     android:orientation="vertical">
 
     <android.support.design.widget.NavigationView
```

Comment:
How do we know the heights of these items are guaranteed to be 48dp? I searched the source and there appears to be a `listPreferredItemHeight`, a `*HeightSmall`, and a `*HeightLarge`. HeightSmall is what we want - how we know it'll get used?

Similarly, how do we know the text color, size, etc. will be what we want them to be? fwiw, there [appears to be some `itemText*` attributes](https://developer.android.com/reference/android/support/design/widget/NavigationView.html#attr_android.support.design:itemTextAppearance) - maybe we should use those?

Reply:
will be clearly set in my next commit